test/e2e/util.go

```diff
@@ -2098,7 +2098,7 @@ func NewHostExecPodSpec(ns, name string) *api.Pod {
 	pod := &api.Pod{
 		TypeMeta: unversioned.TypeMeta{
 			Kind:       "Pod",
-			APIVersion: latest.GroupOrDie("").GroupVersion.Version,
```

Comment:
In particular, this transformation is really problematic. The old code asked specifically about the "" group. The new code asks for some preferred group's version. The old code is much preferable because it's explicit what group the code is working with.